src/frontend/web_application/src/components/Icon/index.jsx

```diff
@@ -80,22 +82,42 @@ export const typeAssoc = {
   'window-maximize': 'fa fa-window-maximize',
 };
 
+const svgGlyphs = {
+  mastodon: (
+    <svg xmlns="http://www.w3.org/2000/svg" width="61.076954mm" height="65.47831mm" viewBox="0 0 216.4144 232.00976">
```

Comment:
you can use https://jakearchibald.github.io/svgomg/ to optimize the markup.

In a futur refactor it would be better to load svg files instead of having it directly into the client, like in [ProviderIcon](https://github.com/CaliOpen/Caliopen/blob/develop/src/frontend/web_application/src/modules/remoteIdentity/components/ProviderIcon/index.jsx)

Reply:
I searched for a solution like this (external svg files), it may need tweaks in webpack config.
And I'm not satisfied with svg for icons, as it does not color as a font does, and makes an unjustified exception for mastodon icon.